enginetest/queries/script_queries.go

```diff
@@ -5897,6 +5897,224 @@ where
 			},
 		},
 	},
+	{
```

Comment:
Can we also add cast tests like:

```
select * from vt where v < cast('def' as varchar(6))
```

etc.

Reply:
I can't cast to varchar, but I can do char(6); will add these!

Reply:
Okay. Are there other types we want to consider here too? Binary, text, etc?